cmd.go

```diff
@@ -22,5 +22,6 @@ const (
 	CMD_GET_URR
 	CMD_GET_BAR
 
+	CMD_GET_REPORT
```

Comment:
New definition should be after CMD_GET_VERSION for compatability with older version of free5GC's UPF.

Reply:
Thanks for notifying.  I have made some changes based on these comments.